interfaces/HO-Portal/src/app/users/users-table/users-table.component.html

```diff
@@ -0,0 +1,95 @@
+<div
+  #proxyScrollbar
+  class="proxy-scrollbar"
+  data-target-scroll-element-selector="datatable-body"
+  data-target-width-element-selector="datatable-scroller"
+>
+  <div class="proxy-scrollbar--content"></div>
+</div>
+<ngx-datatable
+  class="bootstrap data-table--max-height"
+  [rows]="filteredRows"
+  [columnMode]="'force'"
+  [rowHeight]="50"
+  [scrollbarV]="true"
```

Comment:
```suggestion
  [rowHeight]="'auto'"
```

Quickly discussed with PO to tweak the design a little.